server/Dockerfile

```diff
@@ -4,6 +4,7 @@ ENV KEYCLOAK_VERSION 5.0.0
 ENV JDBC_POSTGRES_VERSION 42.2.5
 ENV JDBC_MYSQL_VERSION 5.1.46
 ENV JDBC_MARIADB_VERSION 2.2.3
+ENV JDBC_MSSQL_VERSION 7.0.0.jre8
```

Comment:
Could you please update it to the latest version? See https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc

Reply:
Upgraded to the latest JRE8 driver.